README.md

```diff
@@ -80,7 +80,7 @@ Most basic use:
 
 ```
 editor = RichEditorView(frame: view.bounds)
-editor.html = "<h1>Jesus is God.</h1> He died for our sins and rose from the dead by His own power. Repent and believe the gospel!"
```

Comment:
Please restore this readme text. I didn't mind that you changed the demo text in the controller.